osmaxx-symbology/OSMaxx_point_symbols/svg/special_coniferous.svg

```diff
@@ -12,12 +12,12 @@
    width="352.78mm"
    height="352.78mm"
    viewBox="0 0 1250.0079 1250.0079"
-   id="svg23818"
+   id="svg24477"
    version="1.1"
    inkscape:version="0.91 r13725"
-   sodipodi:docname="special_coniferous.svg">
+   sodipodi:docname="special_deciduous.svg">
```

Comment:
Dear Inkscape: Why-oh-why must a file know its own filename? :-1:
Well, OK, it's probably the document name (which most likely no one of us did explicitly set) ...

@hixi Should these be made to match the new (switched to be correct) file names?